MaeBlock.hpp

```diff
@@ -8,6 +8,7 @@
 #include <string>
 #include <utility>
 
+#include "MaeConstants.hpp"
```

Comment:
wait - why does this change require the constants?

Reply:
oh.. because I originally was not including atom/bond blocks but then decided to return the names of all blocks. Updated!